.travis.yml

```diff
@@ -8,6 +8,10 @@ env:
         - CONTAINER_IMAGE=docker.io/nmstate/centos8-nmstate-dev
           testflags="--test-type integ_tier1 --pytest-args='-x'
               --copr networkmanager/NetworkManager-1.22-git"
+        - CONTAINER_IMAGE=docker.io/nmstate/centos8-nmstate-dev
+          testflags="--test-type integ_tier1 --pytest-args='-x'
+              --copr networkmanager/NetworkManager-1.22-git
+              --customize='dnf remove openvswitch2.11 python3-openvswitch2.11;'"
```

Comment:
Can you explicitly install ovs 2.13 here?

Reply:
In order to install ovs 2.13, I need to remove 2.11 first. If not it will generate conflicts between the package files.